infra/scalingo/get_last_backup_from_scalingo_test.py

```diff
@@ -0,0 +1,118 @@
+from typing import Dict, List
```

Comment:
attention au nom de fichier qui termine par _test `get_last_backup_from_scalingo_test.py`

Reply:
Bon, c'est un test, where is the code ^^ ?